lib/Search/FileSearch/FileSearcher.php

```diff
@@ -21,13 +21,31 @@ class FileSearcher extends TNTSearch {
 		'storage' => ''
 	];
 
+	public const SUPPORTED_LANGUAGES = [
+		'ar' => 'Arabic',
+		'cr' => 'Croatian',
+		'fr' => 'French',
+		'de' => 'German',
+		'en' => 'Porter',
+		'it' => 'Italian',
+		'lv' => 'Latvian',
+		'pl' => 'Polish',
+		'pt' => 'Portuguese',
+		'ru' => 'Russian',
+		'uk' => 'Ukrainian',
+	];
+
+	private const UNSUPPORTED_LANGUAGE = 'No';
```

Comment:
Shouldn't this be some string that is less likely to be similar to `no` (for "Norwegian")?

Reply:
Ok, thanks for the pointer. Then the PR is good to go :blush: Thanks for the contribution @Koc!